### PR TITLE
[WIP] Changed filter-popover to modal. 

### PR DESCRIPTION
**Pull request by philbu**
BUG: Map seems to change modal,view or nav-Controller

@WoH

### CODE REVIEW

**Comment by philbu** [comment]
Now the modal only appears when the map moves/updates otherwise it's blocking all of our content without being visible.

File: ionic2/app/pages/map/map.page.ts [hunk 1]
```diff
@@ -72,9 +72,9 @@ export class MapPage {
   }
 
   showFilterPopover($event?): void {
-    let popover = this.popoverCtrl.create(FilterPopoverComponent);
-    popover.present({
+    let modal = this.modalCtrl.create(FilterPopoverComponent);
+    modal.present(/*{
       ev: $event
-    });
+    }*/);
```
Comment (MajorBreakfast) [inline ionic2/app/pages/map/map.page.ts:78]: A remnant from debugging I guess.

File: ionic2/app/components/filter-popover/filter-popover.component.ts [hunk 2]
```diff
@@ -9,18 +9,7 @@ import { FilterService } from '../../services/filter.service';
...
-        <ion-segment-button value="time">Time</ion-segment-button>
-        <ion-segment-button value="pokemon">Pokemon</ion-segment-button>
-      </ion-segment>
-    </ion-toolbar>
-    <div [ngSwitch]="currentTab">
-      <poke-filter-time-tab *ngSwitchCase="'time'"></poke-filter-time-tab>
-      <poke-filter-pokemon-tab *ngSwitchCase="'pokemon'"></poke-filter-pokemon-tab>
-    </div>
-    `,
+  template: require('./filter-popover.component.html'),
```
Comment (MajorBreakfast) [inline ionic2/app/components/filter-popover/filter-popover.component.ts:12]: I didn't see that that is in your PR. I've added that already. But good idea! :)

**Comment by philbu** [comment]
I think this can be closed (not merged). We fixed the popover and don't need the modal right now

**Comment by WoH** [comment]
I agree

**Comment by AlexanderLill** [comment]
Can this branch be deleted then @philbu ?

**Pull request closed without merging**